Managers/CardManager.gd

```diff
@@ -0,0 +1,25 @@
+extends Node
+## Global AutoLoaded class that allows you to get the currently queued card from anywhere.
+##
+## TODO could potentially use this for global drawing of cards.
+
+
+signal successful_card_play(card: CardWorld)
+
+var queued_card: CardWorld = null
+
+
+func is_card_queued() -> bool:
+	return queued_card != null
+
+
+func set_queued_card(card: CardWorld) -> void:
+	if card == null:
```

Comment:
The idea is you call set_queued_card(null) if you want to unqueue the currently queued card

Reply:
What I mean is queued_card = card already does that if you send in null so technically this if shouldn't be needed - unless there is some other reason or if it's there just to make it more clear I guess

Reply:
Oh yeah, true. I guess this makes more sense in my brain

Reply:
it's true that's it's strange to have `if card == null : queued_card = null`

Reply:
I'll just simplify it to
```
func set_queued_card(card: CardWorld) -> void:
	queued_card = card
```

Reply:
I'm even wondering what is the point of a having a function if it just does `queued_card = card`. Will it be used for a more developed purpose later ?

Reply:
I figured it's a pretty important game event, so we may want other functionality here later. Could fire a signal here that other objects could react to, for example

Reply:
queued_card is currently part of CardManager but it's mostly accessed and used from CardContainer so having a setter for it makes sense, unless we would want to rework how that entire part works or get rid of manager entirely

Reply:
My thinking is we may want to access queued_card from all over the project which is why it's in the globally accessible manager. I think a lot of this will change though when we start on draw pile/discard pile

Reply:
You are right, it might interact with other things later, let's leave it like that